OpenRA.Mods.Common/Traits/Harvester.cs

```diff
@@ -270,7 +281,7 @@ public Order IssueOrder(Actor self, IOrderTargeter order, Target target, bool qu
 
 		public string VoicePhraseForOrder(Actor self, Order order)
 		{
-			return (order.OrderString == "Harvest" || (order.OrderString == "Deliver" && !IsEmpty)) ? "Move" : null;
+			return (order.OrderString == "Harvest" || (order.OrderString == "Deliver" && !IsEmpty)) ? info.Voice : null;
```

Comment:
For the moment - yes. Looks like it won't be any problem separating them, but I'll leave that up to @Mailaender, as this PR isn't small or simple, and that's not really related.